### ISSUE TITLE
support brew services start materialized

### ISSUE THREAD

**Issue by rjnn** [MaterializeInc/homebrew-materialize#21]
optimize for quick and dirty laptop development, i.e. single threaded, running on the default port, no security.

it makes the demos *much* snappier.

```
brew install MaterializeInc/materialize/materialized
brew services start materialized
psql -h localhost -p 6975 -d materialize
> 
```

**Comment by rjnn** [comment]
https://github.com/cockroachdb/homebrew-tap/blob/master/Formula/cockroach.rb#L35-L62

**Comment by rjnn** [comment]
can also use their caveat format (https://github.com/cockroachdb/homebrew-tap/blob/master/Formula/cockroach.rb#L21-L35); dt definitely thought about the most ergonomic way to present the caveat and when.

**Comment by benesch** [comment]
dt? Ahem! https://github.com/cockroachdb/homebrew-tap/commit/a90ee3136cdc31ed54400de441ae43539e7ed40e

**Comment by rjnn** [comment]
😬 damn, you really are everywhere, my bad!

**Comment by benesch** [comment]
Haha, sorry, couldn’t resist. I think we can probably not mention caveats;
we mentioned them at Cockroach mostly because running a single node
insecurely was nonstandard. But that is in fact the only way to run
materialized, so nothing surprising about what the brew formula will do!

**Comment by rjnn** [comment]
alright. lets do it!

**Comment by benesch** [comment]
Finally shipped in v0.4.1.
